src/test/datascience/interactiveWindow.vscode.common.test.ts

```diff
@@ -238,52 +223,6 @@ for i in range(10):
         );
     });
 
-    test('LiveLossPlot', async () => {
-        const code = `from time import sleep
-import numpy as np
-
-from livelossplot import PlotLosses
-liveplot = PlotLosses()
-
-for i in range(10):
-    liveplot.update({
-        'accuracy': 1 - np.random.rand() / (i + 2.),
-        'val_accuracy': 1 - np.random.rand() / (i + 0.5),
-        'mse': 1. / (i + 2.),
-        'val_mse': 1. / (i + 0.5)
-    })
-    liveplot.draw()
-    sleep(0.1)`;
-        const interactiveWindow = await createStandaloneInteractiveWindow(interactiveWindowProvider);
-        await runInteractiveWindowInput(code, interactiveWindow, 1);
-
-        const codeCell = await waitForLastCellToComplete(interactiveWindow);
-        const output = codeCell?.outputs[0];
-        assert.ok(output?.items[0].mime === 'image/png', 'No png output found');
-        assert.ok(
-            output?.metadata?.outputType === 'display_data',
-            `Expected metadata.outputType to be 'display_data' but got ${output?.metadata?.outputType}`
-        );
-    });
-
-    // Create 3 cells. Last cell should update the second
-    test('Update display data', async () => {
```

Comment:
We already have tests for this,
I'm taking this out as I dont think IW should focus on testing core functionality in the Kernel.
Just slows tests

Reply:
@amunger  @rebornix /cc